aspnetcore/blazor/tooling.md

```diff
@@ -581,10 +613,10 @@ To disable SIMD, for example when targeting old browsers (on mobile devices), ad
 </PropertyGroup>
 ```
 
-<!-- UPDATE 8.0 Remove note when https://github.com/dotnet/runtime/issues/89302 is fixed -->
+<!-- UPDATE 9.0 Remove note when https://github.com/dotnet/runtime/issues/89302 is fixed -->
 
 > [!IMPORTANT]
-> During 8.0 preview releases, disabling SIMD isn't supported. For more information, see [SIMD cannot be disabled (dotnet/runtime #89302)](https://github.com/dotnet/runtime/issues/89302).
+> Disabling SIMD isn't supported. For more information, see [SIMD cannot be disabled (dotnet/runtime #89302)](https://github.com/dotnet/runtime/issues/89302).
```

Comment:
If disabling SIMD currently isn't supported should we remove this section? Why tell the user how to disable it and then immediate say it can't be disabled?